user_docs/en/userGuide.t2t

```diff
@@ -1074,6 +1074,32 @@ When you scroll the display, you will then be able to read the rest of the word.
 
 Enabling this may allow for more fluent reading, but generally requires you to scroll the display more.
 
+==== Focus context presentation ====
+This option allows you to choose what context information NVDA will show on the braille display when an object gets focus.
+Context information (or focus ancestry) refers to the hierarchy of objects containing the focus.

```

Comment:
I still wonder whether we should drop the term "focus ancestry" here. I'm concerned it will just confuse most users and it's certainly more technical than "context information".